tests/Core/Util/Common/PrepareForOutputTest.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Tests for the \PHP_CodeSniffer\Util\Common::prepareForOutput() method.
+ *
+ * @author  Bartosz Dziewoński <dziewonski@fastmail.fm>
```

Comment:
I wish I didn't have to be credited here, since this is a trivial contribution. However, the checks complained when I tried to omit this line. Is it possible to avoid this?

Reply:
Thank you! Done.